src/utils/constants.ts

```diff
@@ -38,6 +38,7 @@ export default {
   ERROR_APPLICATION_SUBMIT: 'Problem submitting application',
   ERROR_APPLICATIONS_LIST: 'Problem loading applications list',
   ERROR_REVIEW_APPLICATION: 'Problem loading application for review',
+  ERROR_GENERIC: 'Something went wrong',
   ERROR_REVIEW_PAGE: 'Problem loading review',
   ERROR_REVIEW_OVERVIEW: 'Problem loading review homepage',
   ERROR_REVIEW_SECTION_ROW: 'Problem loading review section row',
```

Comment:
```suggestion
  ERROR_GENERIC: 'Something went wrong',
  ERROR_REVIEW_PAGE: 'Problem loading review',
  ERROR_REVIEW_OVERVIEW: 'Problem loading review homepage',
```

Other error strings to be replaced by Generic error (and removed from here) as part of changes done in #369 - Already added a comment to remember about doing this in there)

Reply:
I've adde a comment, here; https://github.com/openmsupply/application-manager-web-app/issues/369#issuecomment-800704948, basically i noticed wrong error string was being used, and changed to generic (didn't really want to change all of them to correct ones, i think we should still use specific errors)

Reply:
Oh! I think I haven't get the idea correctly sorry! I'll ament to my comment on the other issue

Reply:
You can already remove not in use strings `ERROR_REVIEW_APPLICATION` and `ERROR_REVIEW_SECTION_ROW`. But not a big requirement anyway

Reply:
I was wondering if we can continue that discussion after lunch, and sum it up here: https://github.com/openmsupply/application-manager-web-app/issues/369